core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -239,26 +357,34 @@ object JsonAST {
       if (f.isDefinedAt(x)) f(x) else x
     }
 
-    /** Return a new JValue resulting from applying the given partial function <code>f</code>
-     * to each value in JSON.
-     * <p>
-     * Example:<pre>
-      * JArray(JInt(1) :: JInt(2) :: Nil) transform { case JInt(x) => JInt(x+1) }
-      * </pre>
-      */
+    /**
+     * Return a new `JValue` resulting from applying the given partial function to each value.
+     *
+     * Example:
+     *
+     * {{{
+     * JArray(JInt(1) :: JInt(2) :: Nil) transform {
+     *   case JInt(x) =>
+     *     JInt(x+1)
+     * }
+     * }}}
+     */
     def transform(f: PartialFunction[JValue, JValue]): JValue = map { x =>
       if (f.isDefinedAt(x)) f(x) else x
     }
 
-    /** Return a new JValue resulting from replacing the value at the specified field
-      * path with the replacement value provided. This has no effect if the path is empty
-      * or if the value is not a JObject instance.
-      * <p>
-      * Example:<pre>
-      * JObject(List(JField("foo", JObject(List(JField("bar", JInt(1))))))).replace("foo" :: "bar" :: Nil, JString("baz"))
-      * // returns JObject(List(JField("foo", JObject(List(JField("bar", JString("baz")))))))
-      * </pre>
-      */
+    /**
+     * Return a new JValue resulting from replacing the value at the specified field
+     * path with the replacement value provided. This has no effect if the path is empty
+     * or if the value is not a `[[JObject]]` instance.
```

Comment:
Maybe it might be clearer to say “this `JValue` will be returned unchanged if the path is empty or if this `JValue` is not a `[[JObject]]` instance”?